public-ui/src/app/data-browser/charts/chart-survey-answers/react-survey-answer-chart.component.tsx

```diff
@@ -0,0 +1,275 @@
+    newBaseOptions.xAxis.categories = categoryArr;
+    newBaseOptions.series = chartSeries;
+    newBaseOptions.yAxis.title.text = 'Participant Count';
+    newBaseOptions.yAxis.title.style.fontSize = '16px';
+    newBaseOptions.xAxis.labels.style.fontSize = '14px';
+    newBaseOptions.yAxis.labels.style.fontSize = '16px';
+    newBaseOptions.yAxis.title.style.padding = '1rem';
+    newBaseOptions.plotOptions.column.stacking = 'normal';
+    newBaseOptions.plotOptions.column.colorByPoint = false;
+    newBaseOptions.plotOptions.column.pointWidth = 50;
+    newBaseOptions.plotOptions.column.borderWidth = 0;
+    newBaseOptions.plotOptions.series.animation = {
+        duration: 0
+    };
+    newBaseOptions.plotOptions.series.fontSize = '14px';
+    newBaseOptions.legend = {
+          enabled: false
+    };
+    newBaseOptions.tooltip.followPointer = true;
+    newBaseOptions.tooltip.useHTML = true;
+    newBaseOptions.tooltip.backgroundColor = 'transparent';
+    newBaseOptions.tooltip.formatter = function() {
+        const count = (this.point.y <= 20) ? '&le; 20' : this.point.y;
+        const percentage = ((count / this.point.total) * 100).toFixed();
+        this.point.toolTipHelpText = `
+            <div class="survey-answer-tooltip">
+            <strong>${this.point.series.name}</strong>
+            <span>${count} Participants </span>
+            <span>${percentage}% of all participants who took this version of survey</span>
+            <span>${this.point.total} Total </div></span>`;
+        return this.point.toolTipHelpText;
+    };
+    newBaseOptions.colors = colors;
+    this.setState({options: newBaseOptions});
+  }
+
+  render() {
+      const {options, answerChartInfo} = this.state;
+      return <React.Fragment>
+        <style>{cssStyle}</style>
+        <div className='answer-chart-layout' style={styles.answerChartLayout}>
+        {options && <HighchartsReact highcharts={highCharts} options={options} className='standard-chart' style={styles.standardChart}
+        updateArgs={[true]}/>}
+        <div className='legend' style={styles.legend}>
+                <div className='legend-inner' style={styles.legendInner}>
+                    {
+                        answerChartInfo.map((answer, index) => {
+                        const colorStyle = {color : answer.color};
+                        return <div className='legend-item' key={index} style={styles.legendItem}>
+                        <span><i className='fas fa-circle' style={colorStyle}></i></span>
+                                            <span> {answer.answerValue}</span>
+                        </div>;
+                        })
+                    }
+                </div>
+            </div>
+        </div>
+      </React.Fragment>;
+    }
+}
+
+@Component({
+  selector: 'app-survey-answer-chart-react',
+  template: `<span #root></span>`,
+  styleUrls: ['./chart-survey-answers.component.css'],
+  encapsulation: ViewEncapsulation.None,
```

Comment:
Can we move any styles we're using from this file into this component? Ideally, we would drop these two properties when converting.

Reply:
Sorry forgot to remove them from here. These are already in the react styles.